main.js

```diff
@@ -178,9 +178,39 @@ function logActivity() {
      }
  }
    saveCard();
+   clearTimerSection();
   }
 
-
+  function clearTimerSection() {
+    containerTimer.innerHTML = `<button class="create-new-activity">CREATE A NEW ACTIVITY</button>`;
+    mainTitle.innerText = 'Completed Activity';
+    var createNewActivity = document.querySelector('.create-new-activity');
+    createNewActivity.addEventListener('click', goHome);
```

Comment:
Could we clean this up by doing document.querySelector('.create-new-activity').addEventListener('click', goHome);

Reply:
Ended up finding a way around it by putting the button in the html file to start and attached the event listener directly to it at the top of main js